docs/release/release_0_4_14.md

```diff
@@ -0,0 +1,68 @@
+# napari 0.4.14
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
+
+This release adds the ability to convert shapes layers to labels layers in a context menu available on right clicking the layer (#3978). Our vectors layer has gained the 
```

Comment:
```suggestion
This release adds the ability to convert a shapes layer to a labels layer from a context menu available on right clicking the layer list entry (#3978). Our vectors layer has gained the 
```

Optional: I found the initial wording a little hard to parse. Consider the suggestion, but feel to ignore.